addons/l10n_br_website_sale/models/website.py

```diff
@@ -8,6 +8,6 @@ class Website(models.Model):
     @api.model_create_multi
     def create(self, vals_list):
         for website in vals_list:
-            if self.env['res.company'].browse(website['company_id']).country_code == "BR":
+            if website.get('company_id') and self.env['res.company'].browse(website['company_id']).country_code == "BR":
                 website['show_line_subtotals_tax_selection'] = 'tax_included'
```

Comment:
Ideally, it should be a `setdefault` to avoid overwriting the create value if provided (E.g. you duplicate a website which was already configured to be tax_excluded, the config should not magically change)